driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -724,6 +694,26 @@ public void waitForDown(int node) {
         TestUtils.waitUntilPortIsDown(addressOfNode(node));
     }
 
+    @Override
+    public ProtocolVersion getDesiredProtocolVersion() {
+        VersionNumber version = getCassandraVersion();
```

Comment:
For now the Cassandra version (which is the compatible cassandra version when DSE version is provided) is adequate for determining the protocol version.